mappings/net/minecraft/world/biome/BiomeEffects.mapping

```diff
@@ -57,4 +60,8 @@ CLASS net/minecraft/class_4763 net/minecraft/world/biome/BiomeEffects
 		METHOD method_24942 loopSound (Lnet/minecraft/class_3414;)Lnet/minecraft/class_4763$class_4764;
 			ARG 1 sound
 		METHOD method_24943 moodSound (Lnet/minecraft/class_4968;)Lnet/minecraft/class_4763$class_4764;
+			ARG 1 moodSound
 		METHOD method_24944 additionsSound (Lnet/minecraft/class_4967;)Lnet/minecraft/class_4763$class_4764;
+			ARG 1 additionsSound
+		METHOD method_27346 musicSound (Lnet/minecraft/class_5195;)Lnet/minecraft/class_4763$class_4764;
```

Comment:
actually, can this just be `music`? The sound seems a bit redundant :P

Reply:
sure